lib/project.js

```diff
@@ -25,8 +25,8 @@ Project.prototype = {
   version: Project$version,
   nameVer: Project$nameVer,
   persist: Project$persist,
-  optionalDep: Project$optionalDep,
-  script: Project$script,
+  optionalDep: getterSetter('optionalDependencies'),
+  script: getterSetter('scripts'),
```

Comment:
Naming is hard. Any suggestions on better names? I suppose reversing them would be a decent start.